src/axis/axis.ts

```diff
@@ -1342,11 +1348,14 @@ export module LabelLayoutStrategy {
             // Reset style of text node
             node
                 .style("text-anchor", "middle")
-                .attr({
-                    "dx": "0em",
-                    "dy": "1em",
-                    "transform": "rotate(0)"
-                });
+                .attr(
+                    "dx", "0em")
+                .attr(
+                    "dy", "1em"
+                )
+                .attr(
+                    "transform", "rotate(0)"
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line